src/components/Pages/MyAssets/Collections/CollectionsView.tsx

```diff
@@ -4,12 +4,12 @@ import { styled } from 'styled-components';
 import Title from '@common/Title.tsx';
 import ViewAs from '@common/ViewAs.tsx';
 
-import { ViewAsOptions, ViewType, defaultUiSettings } from '@helpers/config.ts';
+import { ViewAsOptions, ViewType } from '@helpers/config.ts';
 import { UiSettings } from '@helpers/interfaces.ts';
 import { SContentBlockContainer } from '@helpers/reusableStyles.ts';
 
 import { useLoadCollectionsMetadata } from '@hooks/useLoadCollectionsMetadata.ts';
-import { useLocalStorage } from '@hooks/useLocalStorage.ts';
+import { LS_KEY_UI, defaultUiSettings, useLocalStorage } from '@hooks/useLocalStorage.ts';
```

Comment:
I would name it as KEY_UI_SETTINGS, or UI_SETTINGS_KEY

Reply:
and `defaultUiSettings` => `defaultUISettings`

Reply:
Regarding first one I changed to LS_KEY_UI_SETTINGS, LS_KEY is the constant part here, it tells that the key is specifically from local storage

regarding defaultUiSettings I'll leave it as is, it's better to be consistent with camel case and treat abbreviations/acronyms as if they were lowercase words, had a very nasty bug once in a project, because of some automated normalisation tool in AngularJS

Reply:
discussed this, updated accordingly